com.cubrid.common.ui/src/com/cubrid/common/ui/common/preference/JdbcManageComposite.java

```diff
@@ -295,20 +295,15 @@ public void run() {
 					if (taskExec.isSuccess()) {
 						List<String> downloadedList = task.getDriverList();
 						if (downloadedList != null && downloadedList.size() > 0) {
-							downloadedFiles = downloadedList.toString();
+							downloadedFiles = downloadedList.toString().replace(',', '\n');
```

Comment:
@paulgheorghecristian 

Looks good for me. :)
By the way, it seems we can change it like the following so that we have apache commons lang 2.x library in this plugin.
`downloadedFiles = StringUtils.join(downloadedList, '\n');`